### PR TITLE
Added Extensions not showing up in Catalogues FAQ

### PR DESCRIPTION
**Pull request by joseph619**

### CODE REVIEW

File: src/help/faq/extensions.md [hunk 1]
```diff
@@ -88,6 +88,9 @@ Be aware that **Batoto** is deprecated so you will need to migrate from that sou
 Read more about Batoto shutting down [here](https://goboiano.com/manga-scanlation-site-batoto-to-close-down/)
 :::
 
+### ### Why are my extensions not showing up in Catalogues?
+If you have freshly installed/reinstalled Tachiyomi app, the installed extensions won't show up in `Catalogues` until you enable them. To do this, go to **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)**→**[<MaterialIcon icon-name="settings_applications"/> Sources](/help/guides/catalogues/#sources)** . After that, enable the language of your extensions (such as All, English, Spanish etc) and then enable the sources you want to see in `Catalogues`.
```
Comment (Soitora) [inline src/help/faq/extensions.md]: Can you try following what I did more closely on the other sections?

Comment (joseph619) [reply src/help/faq/extensions.md]: What do you want me to change?

**Comment by Soitora** [comment]
@joseph619 This is saying that extensions don't show up in Catalogues until enabled, but I've never had that happen to me? When was this changed?

**Comment by joseph619** [comment]
I think it happens when you install the app new (like when you're migrating), but Extensions still exist so you have to go to Catalogues and enable the source.

**Comment by Soitora** [comment]
Alright, so a reinstall and not just a normal first-install?

**Comment by joseph619** [comment]
Yes, this is also valid for migrating to other versions I think, such as Dev or the forks.

**Comment by joseph619** [comment]
I fixed it to reinstalled

**Comment by Soitora** [comment]
First of all, line #91 has duplicate header symbols.

Also, can you try getting it slimmed down to have it state the basic info then the guide in a guide section like this?: 
image

**Comment by joseph619** [comment]
Done, I trimmed down and made it more concise.

File: src/help/faq/extensions.md [hunk 2]
```diff
@@ -88,6 +88,13 @@ Be aware that **Batoto** is deprecated so you will need to migrate from that sou
 Read more about Batoto shutting down [here](https://goboiano.com/manga-scanlation-site-batoto-to-close-down/)
 :::
 
+### Why are my extensions not showing up in Catalogues?
+If you have freshly reinstalled Tachiyomi app, the installed extensions won't show up in `Catalogues` until you enable them.
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
If you've freshly reinstalled the **Tachiyomi** app, the installed extensions might not show up in **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)** until you enable them. 
```

File: src/help/faq/extensions.md [hunk 3]
```diff
@@ -88,6 +88,13 @@ Be aware that **Batoto** is deprecated so you will need to migrate from that sou
 Read more about Batoto shutting down [here](https://goboiano.com/manga-scanlation-site-batoto-to-close-down/)
 :::
 
+### Why are my extensions not showing up in Catalogues?
+If you have freshly reinstalled Tachiyomi app, the installed extensions won't show up in `Catalogues` until you enable them. 
+
+::: guide
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
::: guide GUIDE
```

File: src/help/faq/extensions.md [hunk 4]
```diff
@@ -88,6 +88,13 @@ Be aware that **Batoto** is deprecated so you will need to migrate from that sou
...
+To fix this, go to **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)**→**[<MaterialIcon icon-name="settings_applications"/> Sources](/help/guides/catalogues/#sources)** . After that, enable the language of your extensions  and then enable the desired sources.
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
To fix this, go to **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)**→**[<MaterialIcon icon-name="settings"/> Sources](/help/guides/catalogues/#sources)**. Now you can proceed to enabling your desired sources.
```

File: src/help/faq/extensions.md (hunk 2 above)
Comment (joseph619) [reply src/help/faq/extensions.md]: Made all requested changes

**Comment by Soitora** [comment]
You haven't done all changes requested yet. I recommend you use the "Commit suggestion" button for instantly adding the suggestions.

**Comment by joseph619** [comment]
It's the dark theme on my Chrome, I couldn't see that button on my screen. I fixed the missing part now. Should be ready

**Comment by Soitora** [comment]
> It's the dark theme on my Chrome, I couldn't see that button on my screen. I fixed the missing part now. Should be ready

I recommend using this:
https://github.com/StylishThemes/GitHub-Dark-Script

Also one final fix since the icon didn't wanna follow the line of the text. After that I'll merge, thanks.

File: src/help/faq/extensions.md [hunk 5]
```diff
@@ -88,6 +88,13 @@ Be aware that **Batoto** is deprecated so you will need to migrate from that sou
 Read more about Batoto shutting down [here](https://goboiano.com/manga-scanlation-site-batoto-to-close-down/)
 :::
 
+### Why are my extensions not showing up in Catalogues?
+If you've freshly reinstalled the **Tachiyomi** app, the installed extensions might not show up in **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)** until you enable them.
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
If you've freshly reinstalled the **Tachiyomi** app, the installed extensions might not show up
in **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)** until you enable them. 
```
Temporary fix for icon not on same-line.

Comment (joseph619) [reply src/help/faq/extensions.md]: So the line break happens after up right? Got it. Also thanks for the dark theme it works well now

**Pull request merged**